examples/geometry/plane_placement_tutorial.py

```diff
@@ -0,0 +1,79 @@
+# bluemira is an integrated inter-disciplinary design tool for future fusion
+# reactors. It incorporates several modules, some of which rely on other
+# codes, to carry out a range of typical conceptual fusion reactor design
+# activities.
+#
+# Copyright (C) 2021 M. Coleman, J. Cook, F. Franza, I.A. Maione, S. McIntosh, J. Morris,
+#                    D. Short
+#
+# bluemira is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# bluemira is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with bluemira; if not, see <https://www.gnu.org/licenses/>.
+"""
+Plotting module examples
+"""
```

Comment:
Unfortunately I was not able to find a simple solution to show the 3D plot in a "realistic" way. Matplotlib just adds the plot of each face on a kind of new layer that overlaps the previous one (intersections are considered). If you think that plot is misleading, we can remove it and, in case, just add a separate plot for each plane. I am open to any better proposals.

Reply:
I thought it might be hard :). I think it is fine for now I was just raising it as something I noticed. Possibly the only thing we could do is comment to say that the non realistic plotting overlaps is a known issue when plotting in mpl. Completely optional!

Reply:
yeah this is just matplotlib not being great at 3-D stuff sadly